pixmappy/PixelMapCollection.py

```diff
@@ -9,8 +9,8 @@
 
 All transformations can be done on arrays of coordinates as well as scalars.
 
-Pixel or world positions are to be numpy arrays of shape (2) or (N,2).  Sky
-positions are astropy.coordinates.SkyCoord objects/arrays.
+Pixel or world positions are to be numpy arrays of shape (2) or (N,2).
+Sky positions are RA, Dec in degrees.
```

Comment:
Fixed.  53d5181

The way I did this PR (from my fork of your repo), you can't made edits until after you merge it.

If you make me a collaborator (Settings tab, Collaborators, Confirm Password, rmjarvis, Add Collaborator), then I can just make a branch on this repo instead, and both of us would be able to make edits before merging.